socket-client.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+const io = require("socket.io-client");
+const port = process.env.PORT || 3000;
+const socket = io(`http://localhost:${port}`);
+const request = require("request");
+
+let bucketName = process.argv[2];
```

Comment:
probably would make sense to have this give an error to the user if they don't pass in `bucketName` or `url`